.gitignore

```diff
@@ -1 +1,3 @@
-_build
\ No newline at end of file
+_build
+.DS_Store
```

Comment:
.DS_Store is a good one to put in a `.gitignore_global` file on your machine so you don't have to put it in every single gitignore.